lib/frodo/profile.rb

```diff
@@ -2,16 +2,32 @@
 
 module Frodo
   class Profile
-    def self.get(id:, token:, version: 'latest')
-      gandalf_url = ENV['GANDALF_URL'].to_s + "/profiles/#{id}/#{version}"
-      new(gandalf_url: gandalf_url, token: token)
+    def self.get(id:, version: 'latest')
+      gandalf_url = ENV['GANDALF_URL'].to_s + "/profiles/#{id}?version=#{version}"
+      new(gandalf_url: gandalf_url)
     end
 
-    def self.instance(location:, token:)
+    def self.instance(location:, token: nil)
       gandalf_url = ENV['GANDALF_URL'].to_s + "/location/#{location.id}/profile"
       new(gandalf_url: gandalf_url, token: token)
     end
 
+    def id
+      data.dig('data', 'id')
+    end
+
+    def version
+      data.dig('data', 'attributes', 'version')
+    end
+
+    def configs
+      data.dig('data', 'attributes', 'data', 'configs')
+    end
+
+    def client_applications
+      data.dig('data', 'attributes', 'data', 'configs')
```

Comment:
Nah bruh.

Tests would've caught this btw

Reply:
Oh I approved but yeah this is a duplicate, this traversal is wrong.